app/views/software_choices_filter.scala.html

```diff
@@ -148,13 +141,12 @@ <h2 class="govuk-heading-m">@messages("softwareChoices.filter.filters")</h2>
                     )
                 )
 
-                <noscript>
-                    <button type="submit" class="govuk-button" data-module="govuk-button">@messages("softwareChoices.filter.filterResults")</button>
-                </noscript>
+                <button type="submit" class="govuk-button" data-module="govuk-button" value="@messages("softwareChoices.filter.filterResults")">@messages("softwareChoices.filter.filterResults")</button>
```

Comment:
The reason why I added the value attribute is because it is used in the ATs


`
    Then("""^I submit the no js apply filters button$"""){ () =>
      FilterSearchPage.findByCss(By.cssSelector("noscript > button")).click() //ATs before change
      FilterSearchPage.findByCss(By.cssSelector("""button[value="Apply filters"]""")).click(). //ATs after change
    }
`